Shared/Samples/Set feature layer rendering mode on scene/SetFeatureLayerRenderingModeOnSceneView.swift

```diff
@@ -0,0 +1,166 @@
+    @State private var isZooming = false
+    
+    /// The viewpoint for the scene.
+    @State private var viewpoint: Viewpoint?
+    
+    /// A Boolean value indicating whether the scene is fully zoomed in.
+    @State private var isZoomedIn = true
+    
+    /// Creates service feature tables using point, polygon, and polyline services.
+    let featureTables: [ServiceFeatureTable] = [URL.pointTable, .polylineTable, .polygonTable].map(ServiceFeatureTable.init(url:))
+    
+    init() {
+        // Iterate through the feature tables and use them to set up feature layers.
+        // Set the rendering mode for either dynamic or static rendering,
+        // and add the feature layers to the scene.
+        for featureTable in featureTables {
+            // Set up the dynamic scene first.
+            let dynamicFeatureLayer = FeatureLayer(featureTable: featureTable)
+            dynamicFeatureLayer.renderingMode = .dynamic
+            dynamicScene.addOperationalLayer(dynamicFeatureLayer)
+            // Then set up the static scene using a clone of the dynamic feature layer.
+            let staticFeatureLayer = dynamicFeatureLayer.clone()
+            staticFeatureLayer.renderingMode = .static
+            staticScene.addOperationalLayer(staticFeatureLayer)
+        }
+        staticScene.initialViewpoint = .zoomedOutViewpoint
+        dynamicScene.initialViewpoint = .zoomedOutViewpoint
+    }
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            SceneViewReader { sceneViewProxy in
+                SceneView(scene: staticScene)
+                    .overlay(alignment: .top) {
+                        Text("Static")
+                            .multilineTextAlignment(.center)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(8)
+                            .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+                    }
+                    .task(id: viewpoint) {
+                        if let viewpoint = viewpoint {
+                            await sceneViewProxy.setViewpoint(viewpoint, duration: 0.5)
+                        }
+                        isZooming = false
+                    }
+            }
+            SceneViewReader { sceneViewProxy in
+                SceneView(scene: dynamicScene)
+                    .overlay(alignment: .top) {
+                        Text("Dynamic")
+                            .multilineTextAlignment(.center)
+                            .frame(maxWidth: .infinity, alignment: .center)
+                            .padding(8)
+                            .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+                    }
+                    .task(id: viewpoint) {
+                        if let viewpoint = viewpoint {
+                            await sceneViewProxy.setViewpoint(viewpoint, duration: 0.5)
```

Comment:
The duration can be a bit longer. I think 5 is good to align with the other map sample.